hs/MakeElements.hs

```diff
@@ -53,6 +53,13 @@ aboutLinks = concatHtml (map stylesheet ["//netdna.bootstrapcdn.com/bootstrap/3.
                                        "static/style/graph/styles.css",
                                        "static/style/common/about.css",
                                        "static/style/common/common.css"])
+drawLinks :: H.Html
+drawLinks = concatHtml (map stylesheet ["//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css",
+                                       "static/style/graph/styles.css",
+                                       "static/style/graph/graph_styles.css",
+                                       "static/style/common/about.css",
+                                       "static/style/common/common.css"])
```

Comment:
@jmatholia you are right, it doesn't exist anymore, this is a fix I actually made last week and it was merged into happstack by David - you might want to merge happstack with your own branch cause a few fixes have been made to it.